server/graphql/types/src/types/program/program_indicator.rs

```diff
@@ -180,6 +185,37 @@ impl IndicatorColumnNode {
     pub async fn column_number(&self) -> i32 {
         self.column.column_number
     }
+
+    pub async fn value(
```

Comment:
can grab Id from here, don't need a separate value_id that calls the same loader just to get id